src/commands/deployment/get.ts

```diff
@@ -142,43 +146,46 @@ export const processOutputFormat = (outputFormat: string): OUTPUT_FORMAT => {
 
 /**
  * Gets a list of deployments for the specified filters
- * @param outputFormat output format: normal | wide | json
- * @param environment release environment, such as Dev, Staging, Prod etc.
- * @param imageTag docker image tag name
- * @param p1Id identifier of the first build pipeline (src to ACR)
- * @param commitId commit Id into the source repo
- * @param service name of the service that was modified
- * @param deploymentId unique identifier for the deployment
+ * @param initObj captures keys and objects during the initialization process
+ * @param values validated command line values
  */
-export const getDeployments = (
+export const getDeployments = async (
```

Comment:
`async` is not needed because this function is already returning a Promise

Reply:
removed